### DISCUSSION TITLE
OpenId Connect

### DISCUSSION THREAD

**Discussion by stelone** [directus/directus#2802] [Archive]
Hello, 

Can Directus be plug on Identy Provider with OpenId Connect ? 

I saw on documentation : 

> NOTE
> We support google SSO with both Google+ (soon to be discontinued) and OpenID Connect.
> https://docs.directus.io/guides/authentication.html#google-sso-setup

But it's the only place, I found this information.

**Comment by alexvdvalk** [comment]
https://docs.directus.io/self-hosted/sso.html#openid
